internal/uuid/uuid.go

```diff
@@ -0,0 +1,31 @@
+package uuid
```

Comment:
nit: Please add the copyright header on new files (just copy it from one of the other opa source files)